src/pages/Museum/MuseumChokha.vue

```diff
@@ -37,8 +49,8 @@
 </template>
 
 <script setup lang="ts">
-import chokhaChallenge from '@/assets/spectrum_data/Museum_Conservation/chokha_mystery_labels_910.webp';
-import chokhaBig from '@/assets/spectrum_data/Museum_Conservation/chokha_mystery_labels_big.webp';
+import chokhaChallenge from '@/assets/spectrum_data/Museum_Conservation/chokha_color_910.webp';
+import chokhaBig from '@/assets/spectrum_data/Museum_Conservation/chokha_color_big.webp';
```

Comment:
```suggestion
import chokhaChallenge from '@/assets/spectrum_data/Museum_Conservation/Chokha_color_910.webp';
import chokhaBig from '@/assets/spectrum_data/Museum_Conservation/Chokha_color_big.webp';
```

My machine was unable to build this PR, but yours clearly could. Which is weird, but I think I know why.

You probably have the repo checked out on your main filesystem which is probably case-insensitive, as Macs (infuriatingly) are by default, and it is recommended to leave them that way to not cause problems.

However, Linux machines like WAPS ARE case-sensitive, and I think will have problems with this. I noticed this problem when downloading all the existing projects. Some of them have the same file with different case on WAPS, which then doesn't transfer right when pulling them down to my machine.

In order to make my local more like the servers, I check out my repos in a separate case-sensitive file system. It shows up under /Volumes/Projects and otherwise works the same as any other folder, but it pays attention to case. I can show you how to do this on Monday if you want :D